lib/Elastica/ResultSet.php

```diff
@@ -62,6 +86,12 @@ public function __construct(Response $response, Query $query, $results)
         $this->_query = $query;
         $this->_response = $response;
         $this->_results = $results;
+
+        $data = $response->getData();
+        $this->_maxScore = isset($data['hits']['max_score']) ? (float) $data['hits']['max_score'] : 0;
```

Comment:
Could we keep this code in the getters and use here the getters instead? It means we use getData() 4 times but that should be ok?